src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -1368,7 +1368,7 @@ MulticopterAttitudeControl::start()
 	_control_task = px4_task_spawn_cmd("mc_att_control",
 					   SCHED_DEFAULT,
 					   SCHED_PRIORITY_MAX - 5,
-					   1700,
+					   2200,
```

Comment:
The actual usage from what I could tell was ~1750B. The current low stack usage warning requires at least 300B+ of free stack space. To get the effectively allocated stack space to be that 2200B is the smallest.